admittance_controller/include/admittance_controller/admittance_rule_impl.hpp

```diff
@@ -120,10 +120,8 @@ controller_interface::return_type AdmittanceRule::reset(const size_t num_joints)
 
 void AdmittanceRule::apply_parameters_update()
 {
-  if (parameter_handler_->is_old(parameters_))
-  {
-    parameters_ = parameter_handler_->get_params();
-  }
+  parameter_handler_->try_get_params(parameters_);
```

Comment:
```suggestion
  parameter_handler_->try_update_params(parameters_);
```

Shouldn't it be `try_update_params`?